content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -791,10 +791,10 @@ Other options:
 
 Prisma Migrate changed from Preview to Generally Available (GA) in 2.19.0.
 
-<Admonition type="warning">
+<Admonition type="info">
 
-**MongoDB not supported** <br />
-Prisma Migrate does not currently support the [MongoDB connector](/concepts/database-connectors/mongodb).
+**Does not apply for MongoDB** <br />
+Instead of `migrate dev` and related commands, [`db push`](/concepts/components/prisma-migrate/db-push) is used for [MongoDB](/concepts/database-connectors/mongodb).
```

Comment:
Point out that Migrate for MongoDB is db push instead of "not supported" warning.